lib/pythontarget.c

```diff
@@ -205,6 +205,53 @@ static PyObject* py_get_logical_time(PyObject *self, PyObject *args) {
     return PyLong_FromLongLong(get_logical_time());
 }
 
+/** 
+ * Return the current tag object.
+ */
+static PyObject* py_get_current_tag(PyObject *self, PyObject *args) {
+    py_tag_t *t = (py_tag_t *) PyType_GenericNew(&TagType, NULL, NULL);
+    if (t == NULL) {
+        return NULL;
+    }
+    t->tag = get_current_tag();
+    return (PyObject *) t;
+}
+
+/**
+ * Compare two tags. Return -1 if the first is less than
+ * the second, 0 if they are equal, and +1 if the first is
+ * greater than the second. A tag is greater than another if
+ * its time is greater or if its time is equal and its microstep
+ * is greater.
+ * @param tag1
+ * @param tag2
+ * @return -1, 0, or 1 depending on the relation.
+ */
+static PyObject* py_compare_tags(PyObject *self, PyObject *args) {
```

Comment:
As I mentioned on lf-lang/lingua-franca#730, we might be able to use [`.tp_richcompare`](https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_richcompare) to offer Pythonic comparisons between tags.

Reply:
that sounds good, I'll do that.